extensions/resource-deployment/notebooks/bdc/2019/CTP3-1/deploy-bdc-aks.ipynb

```diff
@@ -22,8 +22,193 @@
+            "metadata": {}
+        },
+        {
+            "cell_type": "markdown",
+            "source": "### **Check dependencies**",
+            "metadata": {}
+        },
+        {
+            "cell_type": "code",
+            "source": "#Run command helper function\r\ndef run_command():\r\n    print(\"Executing: \" + cmd)\r\n    !{cmd}\r\n    if _exit_code != 0:\r\n        raise SystemExit(f'Shell command:\\n\\n\\t{cmd}\\n\\nreturned non-zero exit code: ' + str(_exit_code) + '.\\n')\r\n    print(f'Successfully executed: {cmd}')\r\n\r\ncmd = 'az --version'\r\nrun_command()\r\ncmd = 'kubectl version --client=true'\r\nrun_command()\r\ncmd = 'mssqlctl --version'\r\nrun_command()",
+            "metadata": {},
+            "outputs": [],
+            "execution_count": 1
+        },
+        {
+            "cell_type": "markdown",
+            "source": "### **Required information**",
+            "metadata": {}
+        },
+        {
+            "cell_type": "code",
+            "source": "import getpass\ndef get_user_input(input_name, is_password = False):\n    if is_password:\n        user_input = getpass.getpass(prompt = input_name)\n        print(input_name + ': ***********')\n    else:\n        user_input = input(input_name)\n        print(input_name + ': ' + user_input)\n    if user_input == \"\":\n        raise SystemExit(f'{input_name} is required')\n    return user_input;\n\nmssql_cluster_name = get_user_input('Cluster name')\nbdc_controller_username = get_user_input('Controller username')\nbdc_controller_password = get_user_input('Controller password', True)\nbdc_mssql_sa_password = get_user_input('SQL Server password', True)\ndocker_username = get_user_input('Docker username')\ndocker_password = get_user_input('Docker password', True)",
+            "metadata": {},
+            "outputs": [],
+            "execution_count": 2
+        },
+        {
+            "cell_type": "markdown",
+            "source": "### **Azure settings**\n*Subscription ID*: visit <a href=\"https://portal.azure.com/#blade/Microsoft_Azure_Billing/SubscriptionsBlade\">here</a> to find out the subscriptions you can use, if you leave it unspecified, the default subscription will be used.\n\n*VM Size*: visit <a href=\"https://docs.microsoft.com/en-us/azure/virtual-machines/linux/sizes\">here</a> to find out the available VM sizes you could use. \n \n*Region*: visit <a href=\"https://azure.microsoft.com/en-us/global-infrastructure/services/?products=kubernetes-service\">here</a> to find out the Azure regions where the Azure Kubernettes Service is available.",
+            "metadata": {}
+        },
+        {
```

Comment:
Is it good to hardcode the region to eastus? May be we should prompt this for user?

Reply:
this falls into the "default" settings, the idea is to reduce the number of prompts :)